src/test/pipeline/config-local.js

```diff
@@ -0,0 +1,20 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const os = require('os');
+var config = exports;
```

Comment:
Not sure if this does what you mean - config optionally loads config-local to allow overwriting configs locally, but to do that config-local should require the original config module in order to overwrite it, and not update in a local exported object.

Reply:
Line https://github.com/noobaa/noobaa-core/blob/e4c92b74a59f5ba873ed28737989d3ef53893fc5/config.js#L470 will merge local-config into config. If you have config as { a: 1, b:2}, and local_config as { b:3, c:4}, config will be { a:1, b:3, c:4} after the call. 

config-local.js does not need to require config.js.

Reply:
oh you're right, I didn't notice that was added.